lineman/app/components/flash/flash.scss

```diff
@@ -14,6 +14,25 @@
   padding: 15px;
   width: 100%;
   border-radius: 6px;
+  display: table;
+}
+
+.flash-root__message {
+  display: table-cell;
+  padding-right: 10px;
+}
+
+.flash-root__action {
+  display: table-cell;
```

Comment:
Right ok. Update on media  .. we're actively trying to avoid it. I really like display: flex ... I think it's worth using.  I'm ok/happy with this, but I kinda wish it was display: flex and we adopted that everywhere it made sense.